src/repl/scala/tools/nsc/interpreter/PresentationCompilerCompleter.scala

```diff
@@ -33,6 +33,7 @@ class PresentationCompilerCompleter(intp: IMain) extends Completion {
 
     // secret handshakes
     val slashPrint  = """.*// *print *""".r
+    val slashPrintRaw  = """.*// *printRaw *""".r
```

Comment:
It would be natural to `print(Raw)` and `case slashPrint(raw) => if (raw != null) etc`.